binaries/aot_model_compiler.cc

```diff
@@ -198,6 +204,10 @@ int main(int argc, char** argv) {
       c10::DictType::create(c10::StringType::get(), c10::AnyType::get());
   auto compiled_module = torch::jit::detail::codegen_backend_module(
       "nnc", frozen_m, compile_spec, any_dict_ty);
+  auto methods_to_clone = split(',', FLAGS_methods_to_clone);
+  for (const auto& method : methods_to_clone) {
```

Comment:
Hi, can we do this inside the `preprocess` method ? That way when we create a python binding, this logic would automatically get included in torch._C._jit_to_backend("nnc", model, ...) call.